templates/messaging-system/undesired-events/undesired-events.1.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "undesired-events",
+  "properties": {
+    "impression_id": {
+      "description": "An alternative UUID used when client_id is absent",
```

Comment:
This description doesn't seem quite right since there's no client_id field here.

Reply:
Nice catch! Fixed.